src/main/java/iana/command/AddTaskCommand.java

```diff
@@ -6,13 +6,23 @@
 import iana.tasks.Task;
 import iana.tasks.TaskList;
 
+/**
+ * Command that adds a new task to the list.
+ */
 public class AddTaskCommand extends Command {
     private String taskInput;
 
+    /**
+     * Constructor for AddTaskCommand.
+     * @param taskInput the full input that describes the task.
```

Comment:
Do note to leave a spacing between the description and @param for JavaDocs! This applies a couple of methods in your other classes as well!